lib/child-writer/writer-proxy.js

```diff
@@ -157,6 +158,13 @@ return isElevated().then((elevated) => {
       message: 'User did not grant permission.'
     }, () => {
       process.exit(EXIT_CODES.CANCELLED);
+    }).catch({
+      message: 'No polkit authentication agent found.'
+    }, () => {
+      throw errors.createUserError(
+        'No polkit authentication agent found',
+        'Please install a polkit authentication agent for your desktop environment of choice to continue'
```

Comment:
Virtually all user friendly GNU/Linux distributions will include a polkit agent, so if you don't have one then it means you either have a less known distribution, or you are an experienced user that removed it for some reason. In either case, it means that you're an advanced user, so I think its fine.